util/fileUtils.go

```diff
@@ -152,17 +152,24 @@ func IsDir(path string) bool {
 
 // GetSpecFiles returns the list of spec files present at the given path.
 // If the path itself represents a spec file, it returns the same.
-var GetSpecFiles = func(path string) []string {
+var GetSpecFiles = func(paths []string) []string {
 	var specFiles []string
-	if common.DirExists(path) {
-		specFiles = append(specFiles, FindSpecFilesIn(path)...)
-	} else if common.FileExists(path) && IsValidSpecExtension(path) {
-		f, _ := filepath.Abs(path)
-		specFiles = append(specFiles, f)
+	for _, path := range paths {
+		if common.DirExists(path) {
+			specFiles = append(specFiles, FindSpecFilesIn(path)...)
+		} else if common.FileExists(path) && IsValidSpecExtension(path) {
+			f, _ := filepath.Abs(path)
+			specFiles = append(specFiles, f)
+		}
 	}
+
 	return specFiles
 }
 
+var GetSpecFilesFrom = func(path string) [] string{
```

Comment:
You are right about go not supporting overloading. Instead of having two functions, In places where only one path is sent, we could convert it to an array before sending it. 
you can convert a string to array as such `[]string{"something"}` .

Reply:
I had this first, but in my eyes this is a bad design, when you force a caller to follow a certain structure. But this of course is easier, when you have a language that supports overloading. 
I will change that.

Regarding the CLA signing, I did this twice yesterday, with getting a big green checkmark when clicking the button - I can do this again, but if this failed the first two times already ?!

Reply:
> Regarding the CLA signing, I did this twice yesterday

You don't have to do this again.